tetris.py

```diff
@@ -175,12 +192,34 @@ def get_current_block(self):
 
         return coords[self.rotate_counter % len(coords)]
 
+    def coord_extractor(self):
+        coords = []
+        for square in self.landed_blocks:
+            coords.append(square[1])
+        return coords
```

Comment:
If you write it like this, then the person reading the code doesn't have to guess what `square[1]` does:

```suggestion
        return [coords for shape, coords in self.landed_blocks]
```